app.js

```diff
@@ -1,178 +1,182 @@
-        }
+	var data = { username: 'Adekunle'};
+	fs.readFile(__dirname+'/template/recipe-details.html',function(err,data){
+		if(err)
+		{
+			res.send('error occured '+err);
+		}
+		else{
+			res.end(data);
+		}
        
-    });
+	});
 });
 app.get('/favourite-recipes',function(req,res)
 {
-    var data = { username: "Adekunle"};
-    fs.readFile(__dirname+'/template/favourite-recipes.html',function(err,data){
-        if(err)
-        {
-            res.send("error occured "+err);
-        }
-        else{
-            res.end(data);
-        }
+	var data = { username: 'Adekunle'};
+	fs.readFile(__dirname+'/template/favourite-recipes.html',function(err,data){
+		if(err)
+		{
+			res.send('error occured '+err);
+		}
+		else{
+			res.end(data);
+		}
        
-    });
+	});
 });
 app.get('/user-profile',function(req,res)
 {
-    var data = { username: "Adekunle"};
-    fs.readFile(__dirname+'/template/user-profile.html',function(err,data){
-        if(err)
-        {
-            res.send("error occured "+err);
-        }
-        else{
-            res.end(data);
-        }
+	var data = { username: 'Adekunle'};
+	fs.readFile(__dirname+'/template/user-profile.html',function(err,data){
+		if(err)
+		{
+			res.send('error occured '+err);
+		}
+		else{
+			res.end(data);
+		}
        
-    });
+	});
 });
 app.get('/add-recipe',function(req,res)
 {
-    var data = { username: "Adekunle"};
-    fs.readFile(__dirname+'/template/add-recipe.html',function(err,data){
-        if(err)
-        {
-            res.send("error occured "+err);
-        }
-        else{
-            res.end(data);
-        }
+	var data = { username: 'Adekunle'};
+	fs.readFile(__dirname+'/template/add-recipe.html',function(err,data){
+		if(err)
+		{
+			res.send('error occured '+err);
+		}
+		else{
+			res.end(data);
+		}
        
-    });
+	});
 });
-//load fortune library
-var fortune = require('./lib/fortune.js');
 
-//manage test
+
+
+// manage test
 app.use(function(req,res,next){
-res.locals.showTests = app.get('env') !== 'production' && req.query.text ==='1';
-next();
+	res.locals.showTests = app.get('env') !== 'production' && req.query.text ==='1';
+	next();
 });
 
 
-//add startup route
+// add startup route
 app.get('/',function(req,res)
 {
-    var data = { username: "Adekunle"};
-    fs.readFile(__dirname+'/template/home.html',function(err,data){
-        if(err)
-        {
-            res.send("error occured");
-        }
-        else{
-            res.end(data);
-  //res.end(data);
-        }
+	var data = { username: 'Adekunle'};
+	fs.readFile(__dirname+'/template/home.html',function(err,data){
+		if(err)
+		{
+			res.send('error occured');
+		}
+		else{
+			res.end(data);
+			// res.end(data);
+		}
        
-    });
-   // res.render('home',data);
+	});
+	// res.render('home',data);
 });
 
-require(`${__dirname}/server/routes`)(app,fs,sequelize,express);
 
 
 
-//add dashboard route
+
+// add dashboard route
 app.get('/dashboard',function(req,res)
```

Comment:
ES6 arrow functions should be used as suppose to ES5 way

Reply:
oh! My bad. i was rushing to submit the codes for review.. Necessary changes will be made